__tests__/edgeCases.test.jsx

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
These two rules (destructuring-assignment and state-in-constructor) seem a bit too much for me. What do you think about disabling them in the config instead?

Reply:
We can do that. :)